src/app.ts

```diff
@@ -11,8 +21,32 @@ winston.configure({
     ],
 });
 
-import { Matchmaker } from "./matchmaker";
+app.get("/start", bodyParser.json(), (req, res) => {
+    try {
+        winston.info(req.body);
+        mm.start();
+        res.end();
+    } catch (e) {
+        winston.error(e);
+        throw new InternalServerError(e);
+    }
+});
 
-const mm = new Matchmaker();
+app.get("/stop", bodyParser.json(), (req, res) => {
+    try {
+        mm.stop();
+        res.end();
+    } catch (e) {
+        winston.error(e);
+        throw new InternalServerError(e);
+    }
+});
 
-mm.start();
+app.get("/queued", async (req, res) => {
+    res.json(await db.connection("games").where({ status: "queued" }).then(db.rowsToGames));
```

Comment:
What happens then an error is thrown? There should be a .catch that passes the error to the next function

`.catch(next)`

Which is later routed to an error handler (see comment above about error handling)